src/TodoList.js

```diff
@@ -13,16 +18,18 @@ let todoList = [
       title:"Call mom"
     }
   ]; 
+  return(
+    // const listItems = todoList.map(function(item){
+    //     return <li key={item.id}> {item.title}</li>;
+    //   });
 
-const TodoList = () => {
-    const listItems = todoList.map(function(item){
-        return <li key={item.id}> {item.title}</li>;
-      });
-    return(
-      <ul>
-        {listItems}
-      </ul>
-    );
-}
+    <ul>
+      {todoList.map((item) => ( 
+        <TodoListItem key={item.id} todo={item.title} />))     
```

Comment:
here we pass pass only title

`todo={item.title}`

either we should name it as title like 

`title={item.title}`

or we should pass all object

`todo={item}` if we pass object we should also update how we use it in TodoListItem component